src/prefect/filesystems.py

```diff
@@ -572,3 +576,95 @@ async def read_path(self, path: str) -> bytes:
 
     async def write_path(self, path: str, content: bytes) -> str:
         return await self.filesystem.write_path(path=path, content=content)
+
+
+class SMB(ReadableFileSystem, WritableFileSystem):
+    """
+    Store data as a file on a SMB share.
```

Comment:
```suggestion
    Store data as a file on a SMB share.
    
```

Reply:
Adding some linespace here to make sure the markdown renders correctly. It can't hurt, and sometimes the markdown parser pulls adjacent lines up into a single paragraph.